frontend/src/views/Transfers/components/AttachmentList.jsx

```diff
@@ -3,20 +3,20 @@ import BCBox from '@/components/BCBox'
 import AttachFileIcon from '@mui/icons-material/AttachFile'
 import { Typography, List, ListItemButton, ListItemText } from '@mui/material'
 
-export const AttachmentList = ({ attachments }) => {
+export const AttachmentList = ({ attachments = [] }) => {
   return (
     <BCBox mt={2}>
       <Typography variant="h6" color="primary">
         Attachments
       </Typography>
       <List sx={{ maxWidth: '30%' }}>
-        {attachments.map((attachment) => (
+        {attachments.map((attachment, index) => (
           <ListItemButton
             component="a"
-            key={attachment.attachmentID}
+            key={`${attachment.attachmentID}-${index}`}
```

Comment:
Why is this necessary? Is the attachment ID not unique?

Reply:
You're right. Initially, I thought it would be better to make attachments unique in case of duplicates, but it doesn't really make sense. I've reverted the changes. Thanks!